src/prog_algs/metrics/toe_metrics.py

```diff
@@ -0,0 +1,60 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the National Aeronautics and Space Administration. All Rights Reserved.
+
+"""
+This file includes functions for calculating metrics given time of event (ToE)
```

Comment:
Again, pretty clear as is, but maybe "This file includes functions for calculating metrics specific to time of event (ToE) from a single event or multiple events given the same time of prediction"

Reply:
```suggestion
This file includes functions for calculating metrics specific to time of event (ToE) from a single event or multiple events given the same time of prediction
```